gulpfile.js

```diff
@@ -339,6 +342,13 @@ gulp.task('default', function() {
   });
 });
 
+gulp.task('extension', function (done) {
+  console.log();
+  console.log('### Building extensions');
+
+  runSequence('locale', ['firefox', 'chromium'], done);
```

Comment:
According to https://www.npmjs.com/package/run-sequence, an Array means parallel execution. Should we just change that and see if it helps?